Integrations/ArcSightESMv2/ArcSightESMv2.py

```diff
@@ -492,7 +518,9 @@ def get_security_events(event_ids, last_date_range=None):
         events = res_json.get('sev.getSecurityEventsResponse').get('sev.return')
         return events if isinstance(events, list) else [events]
 
-    return_error('Events are empty for some reason. Response Body: {}'.format(res.text))
+    demisto.debug(res.text)
```

Comment:
I think its better to add string that says what is this and then connect the `res.text` content

Reply:
res.text is a huge xml traceback who don't have much meaning to customers, hence it's only logged. also, this is not an error, we will get to this line if there are no base events related to the case.